package.json

```diff
@@ -4,31 +4,15 @@
   "main": "index.js",
   "repository": "git@github.com:mozilla-frontend-infra/perf-goggles.git",
   "author": "Armen Zambrano G. <armenzg@gmail.com>",
-  "license": "MIT",
+  "license": "MPL-2.0",
```

Comment:
I don't see a `main` entry here for what this library runs.

Reply:
Something like ["main": "build/index.js"](https://github.com/taskcluster/taskcluster-client-web/blob/master/package.json#L4)?: